main/src/cgeo/geocaching/log/LogEntry.java

```diff
@@ -208,7 +210,7 @@ public Builder setFound(final int found) {
          * Set {@link LogEntry} friend.
          *
          * @param friend
-         *          {@code true} if this is a log from a friend
+         *          {@code true} if this is a log from current user himself or his friend
```

Comment:
While I appreciate these clarifications I'd rather see them in a separate PR as they, if I understand correctly, have nothing to do with the change as such.